lib/ueberauth/config/config.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Ueberauth.Config do
+  @moduledoc """
+  A json library is required for Ueberauth to operate.
```

Comment:
I would move `json_library/0` into `Ueberauth` module instead of having a `Config` module.

cc: @ueberauth/core